src/components/PolygonAddressInfo.js

```diff
@@ -0,0 +1,88 @@
+/* global IqonHash */
+/* global LoginFileConfig */
+/* global TemplateTags */
+/* global I18n */
+
+class PolygonAddressInfo { // eslint-disable-line no-unused-vars
+    /**
+     * @param {string} address
+     * @param {string} [label]
+     * @param {'none' | 'usdc' | 'unknown'} [logo = 'none']
+     */
+    constructor(address, label, logo = 'none') {
+        this._address = address;
+        this._label = label;
+        this._logo = logo;
+    }
+
+    /**
+     * Inserts this AddressInfo into $el overwriting the original content of $el.
+     * @param {HTMLElement} [$el]
+     * @returns {HTMLElement}
+     */
+    render($el) {
```

Comment:
```suggestion
    renderTo($el) {
```

Should be called `renderTo`, congruent to the naming in `AddressInfo`.